### ISSUE TITLE
Bug -  CSS Foundations : HTML and CSS Codes in the Specificity Section Does Not Get Rendered Properly

### ISSUE THREAD

**Issue by abhilash-mukherjee** [TheOdinProject/curriculum#24355]
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `Bug - location of bug: brief description of bug` format, e.g. `Bug - React section: Knowledge Checks don't link to resource`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Bug:**

I commited a few changes in the "specificity" section of CSS foundation a few days ago. 
It included addition of an example containing some html code.
Link to discussion: https://github.com/TheOdinProject/curriculum/pull/24315#partial-pull-merging
The changes were merged into the project by @fortypercenttitanium but now there seems to be some issue as the html code is missing and the css code are not getting rendered properly. See Below:

![CSS-Foundation-Browser-Image](https://user-images.githubusercontent.com/84319482/177046576-0e305d03-365b-403c-b340-cb2dffd1d4bc.png)
 

**When I checked the code for this section on the Github repo, it looks fine there. See below:**
 

![CSS-Foundation-Code-Image](https://user-images.githubusercontent.com/84319482/177046620-d5f0fc64-abe6-4648-a329-b29b58b85c01.png)

**2. How To Reproduce:**

1. Open the link: https://www.theodinproject.com/lessons/foundations-css-foundations
2. Scroll to the "Specificity" section
3. Html and CSS code not rendered properly

**3. Expected Behavior:**

1. Open the link: https://www.theodinproject.com/lessons/foundations-css-foundations
2. Scroll to the "Specificity" section
3. Html and CSS code SHOULD BE PROPERLY EMBEDDED

**4. Desktop/Device:**

-   Device: 
-   OS: Windows 10
-   Browser: Chrome, Mozilla, Brave
-   Version: 

**5. Additional Information:**

None

**Comment by Tabarna88** [comment]
Commented just 18 minutes ago, amazing! hahahaha I came to raise the same problem but you acted faster than me.

**Comment by fortypercenttitanium** [comment]
@abhilash-mukherjee Thanks for catching this. It was my mistake. Would you be able to fix it today?

**Comment by abhilash-mukherjee** [comment]
Sorry. It's sleep time at my place so, I don't think I would be able to solve it today. But I will surely try to fix it by tomorrow.
But, if its urgent, it can be assigned to someone else.

**Comment by fortypercenttitanium** [comment]
> Okay. Will try to do it ASAP

Within 24 hours would be fine, just let me know if you have problems getting to it.

**Comment by abhilash-mukherjee** [comment]
@fortypercenttitanium Hey. I had a small query.. Maybe a bit silly.. but still will ask:
When I go to the github code [here](https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/css-foundations.md?plain=1), it is rendering fine.

But, its not working properly in the browser. So, can it be at all rectified by making changes in this [code ](https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/css-foundations.md?plain=1). Or, do I need to go to some other code which defines the structure and styling of the website to fix the bugs?

**Comment by fortypercenttitanium** [comment]
> @fortypercenttitanium Hey. I had a small query.. Maybe a bit silly.. but still will ask: When I go to the github code [here](https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/css-foundations.md?plain=1), it is rendering fine.
> 
> But, its not working properly in the browser. So, can it be at all rectified by making changes in this [code ](https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/css-foundations.md?plain=1). Or, do I need to go to some other code which defines the structure and styling of the website to fix the bugs?

Try removing the line break on line 294. It seems to work fine in the other instances when there is no line break between the two blocks.

**Comment by abhilash-mukherjee** [comment]
Okay

**Comment by abhilash-mukherjee** [comment]
@fortypercenttitanium I tried fixing this issue by removing the line break in #24359. I hope it will work now

**Comment by xandora** [comment]
This is still faulty even after the fix in #24359. The fault is caused by insufficient whitespace before the `~~~html` markdown at the start of the code block.

Bugged:
```markdown
Let's take a look at a few quick examples to visualize how specificity works. 
Consider the following HTML and CSS code:
~~~html
<!-- index.html -->

<div class="main">
  <div class="list subsection"></div>
</div>
~~~
```

Corrected:
```markdown
Let's take a look at a few quick examples to visualize how specificity works. 
Consider the following HTML and CSS code:

~~~html
<!-- index.html -->

<div class="main">
  <div class="list subsection"></div>
</div>
~~~
```

**Comment by xandora** [comment]
This has been resolved: https://github.com/TheOdinProject/curriculum/commit/abbe98219cedc8aef5a40faec4ed7ac0eb30a33c
